platform/pulp/platform/fields.py

```diff
@@ -0,0 +1,80 @@
+"""
+Custom django fields provided by pulp platform
+"""
+import json
+
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import six, ugettext_lazy as _
+
+
+class JSONField(models.TextField):
+    """
+    A custom django field to serialize dicts into a text field
+
+    """
+    def from_db_value(self, value, expression, connection, context):
+        """
+        Converts a value as returned by the database to a Python object
+
+        :param value: The value to convert to Python
+        :type value: Any
+
+        :param expression:
+        :param connection:
+        :param context:
+
+        :return: A python representation of value
+        :rtype: Any
+        """
+        if isinstance(value, six.string_types):
+            return self.to_python(value)
+
```

Comment:
I generally don't put whitespace before the return statement. Pep 8 recommends using whitespace sparingly to separate logical blocks of code. It's a preference thing so this is not a required change but I wanted to mention it. It's possible someone would miss the return value since they already see one above.